src/components/emptyState/NoSearchResults.tsx

```diff
@@ -15,11 +13,6 @@ interface NoSearchResultsProps {
 const NoSearchResults: FunctionComponent<NoSearchResultsProps> = ({ clearFilters }) => {
   return (
     <EmptyState variant={EmptyStateVariant.sm}>
-      <EmptyStateHeader
-        titleText="No results found"
-        icon={<EmptyStateIcon icon={SearchIcon} />}
-        headingLevel="h2"
-      />
```

Comment:
Issue: This content still needs to be shown somehow - if the header component is deprecated there needs to be some other way to render this